manifest.js

```diff
@@ -35,6 +83,7 @@ module.exports = Manifest({
   tokenManagementEnabled: true,
```

Comment:
We should be able to remove this field once the latest version of the deno-slack-sdk is cut.